tests/api/v2/datadog/api_logs_test.go

```diff
@@ -0,0 +1,138 @@
+	if err != nil {
+		t.Fatalf("Could not list logs: %v", err)
+	}
+	assert.Equal(200, httpResp.StatusCode)
+	assert.Equal(2, len(response.GetData()))
+	content := response.GetData()[0].GetContent()
+	assert.Equal("test-log-list-1 " + suffix, content.GetMessage())
+
+	content = response.GetData()[1].GetContent()
+	assert.Equal("test-log-list-2 " + suffix, content.GetMessage())
+
+	request.SetSort(datadog.LOGSSORT_TIMESTAMP_DESCENDING)
+
+	response, httpResp, err = Client(ctx).LogsApi.ListLogs(ctx).Body(*request).Execute()
+	if err != nil {
+		t.Fatalf("Could not list logs: %v", err)
+	}
+	assert.Equal(200, httpResp.StatusCode)
+	assert.Equal(2, len(response.GetData()))
+	content = response.GetData()[0].GetContent()
+	assert.Equal("test-log-list-2 " + suffix, content.GetMessage())
+
+	content = response.GetData()[1].GetContent()
+	assert.Equal("test-log-list-1 " + suffix, content.GetMessage())
+
+	// Paging
+	page := datadog.NewLogsListPayloadPage()
+	page.SetLimit(1)
+	request.SetPage(*page)
+	response, httpResp, err = Client(ctx).LogsApi.ListLogs(ctx).Body(*request).Execute()
+	if err != nil {
+		t.Fatalf("Could not list logs: %v", err)
+	}
+
+	assert.Equal(200, httpResp.StatusCode)
+	assert.Equal(1, len(response.GetData()))
+	respMeta := response.GetMeta()
+	respPage := respMeta.GetPage()
+	cursor := respPage.GetAfter()
+	firstId := response.GetData()[0].GetId()
+
+	request.Page.SetCursor(cursor)
+	response, httpResp, err = Client(ctx).LogsApi.ListLogs(ctx).Body(*request).Execute()
+	if err != nil {
+		t.Fatalf("Could not list logs: %v", err)
+	}
+	assert.Equal(200, httpResp.StatusCode)
+	assert.Equal(1, len(response.GetData()))
+	secondId := response.GetData()[0].GetId()
+
+	assert.NotEqual(firstId, secondId)
+}
+
+func sendLogs(ctx context.Context, t *testing.T, suffix string) error {
+	now := tests.ClockFromContext(ctx).Now()
+	source := fmt.Sprintf("go-client-test-%s", suffix)
+	firstMessage := fmt.Sprintf("test-log-list-1 %s", suffix)
+	secondMessage := fmt.Sprintf("test-log-list-2 %s", suffix)
+	hostname := *tests.UniqueEntityName(ctx, t)
+
+	httpLog := fmt.Sprintf(
+		`{"ddsource": "%s", "ddtags": "go,test,list", "hostname": "%s", "message": "{\"timestamp\": %d, \"message\": \"%s\"}"}`,
+		source, hostname, (now.Unix()-1000)*1000, firstMessage,
+	)
+	httpresp, respBody, err := SendRequest(ctx, "POST", "https://http-intake.logs.datadoghq.com/v1/input", []byte(httpLog))
+	if err != nil {
+		return fmt.Errorf("response %s: %v", respBody, err)
+	}
+	if httpresp.StatusCode != 200 {
+		return fmt.Errorf("status=%d response=%v", httpresp.StatusCode, respBody)
+	}
+
+	httpLog = fmt.Sprintf(
+		`{"ddsource": "%s", "ddtags": "go,test,list", "hostname": "%s", "message": "{\"timestamp\": %d, \"message\": \"%s\"}"}`,
+		source, hostname, (now.Unix()-1)*1000, secondMessage,
+	)
+	httpresp, respBody, err = SendRequest(ctx, "POST", "https://http-intake.logs.datadoghq.com/v1/input", []byte(httpLog))
```

Comment:
Can you make it works with `testSite, ok := os.LookupEnv("DD_TEST_SITE")` (see `NewConfiguration ` function in https://github.com/DataDog/datadog-api-client-go/pull/342/files#diff-a040067fa60b707e7e32f31d3d42ef01R62).

Reply:
There are teams that would like to run the test suite with staging.